packages/api-reference/src/features/example-responses/ExampleResponses.vue

```diff
@@ -26,17 +29,16 @@ import ExampleResponseTabList from './ExampleResponseTabList.vue'
  * TODO: copyToClipboard isn't using the right content if there are multiple examples
  */
 
-const { responses } = defineProps<{ responses: Operation['responses'] }>()
+const { responses } = defineProps<{
+  responses: Dereference<OperationObject>['responses']
```

Comment:
```suggestion
  responses: ResponsesObject
```

You can use the ResponsesObject directly here